DOC: Update python as glue doc. (#24133)
numpy/numpy: doc/source/user/c-info.python-as-glue.rst

====================
Using Python as glue
====================

> This was written in 2008 as part of the original
> `Guide to NumPy <https://archive.org/details/NumPyBook>`_ book
> by Travis E. Oliphant and may be out of date.

Reviewer: ```suggestion

.. warning::
   This was written in 2008 as part of the original
   `Guide to NumPy <https://archive.org/details/NumPyBook>`_ book
   by Travis E. Oliphant and is out of date.
```
